scripts/jenkins/jobs-ibs/cloud-mkphyscloud-qa-scenario-1a.yaml

```diff
@@ -0,0 +1,157 @@
+- job:
+    name: 'cloud-mkphyscloud-qa-scenario-1a'
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      <b>This job is managed by JJB! Changes must be done in <a href='https://github.com/SUSE-Cloud/automation/tree/master/scripts/jenkins/jobs-ibs/'>git</a></b>
+
+      This job will redeploy the qa2.cloud with scenario-1a:
+        - 7 nodes
+        - HA with IPMI (3 controllers
+        - shared device for DB and rabbit
+        - swift
+
+      It will wipe all QA2 machines!
+
+    wrappers:
+      - mkphyscloud-qa-common-wrappers
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    parameters:
+      - string:
+          name: cloudsource
+          default: develcloud6
+      - string:
+          name: TESTHEAD
+          default: "1"
+          description: if non-empty, test latest version from Devel:Cloud Stagin
+      - string:
+          name: cct_tests
+          default: features:base
+          description: Run cct
+      - string:
+          name: want_ceph
+          default:
+          description: Install ceph barclamp
+      - string:
+          name: hw_number
+          default: "2"
+          description: Name of the QA cloud server (we have 2,3,4)
+      - string:
+          name: hacloud
+          default: "1"
+          description: By default we do not want HA configured and installed
+      - string:
+          name: clusterconfig
+          default: services=3
+          description: HA configuration for clusters. Make sense only if hacloud=1
+      - string:
+          name: nodenumber
+          default: "7"
+          description: Number of nodes to use. Depends on hw_number
+      - string:
+          name: want_ipmi
+          default: "true"
+      - string:
+          name: runner_url
+          default: https://raw.github.com/SUSE-Cloud/automation/master/scripts/qa_crowbarsetup.sh
+          description: The runner must be qa_crowbarsetup.sh file
+      - string:
+          name: commands
+          default: prepareinstallcrowbar installcrowbar allocate waitcloud setup_aliases batch
+          description: All the steps that needs to be completed to have cloud installed
+      - string:
+          name: scenario_url
+          default: https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml
+          description: Location of scenario yaml file
+      - text:
+          name: UPDATEREPOS
+          default:
+          description: Update repositories (one URL per line)
+      - bool:
+          name: UPDATEBEFOREINSTALL
+          default: false
+          description: add update repos before crowbar install
+      - bool:
+          name: WITHREBOOT
+          default: false
+      - string:
+          name: tempest
+          default:
```

Comment:
Default value should be smoke, description could be: `Specifies what tempest test(s) to run, e.g. smoke or smoke|full or smoke|defcore